src/sentinel.c

```diff
@@ -3331,12 +3377,12 @@ int sentinelSendSlaveOf(sentinelRedisInstance *ri, char *host, int port) {
     ri->pending_commands++;
 
     retval = redisAsyncCommand(ri->cc,
-        sentinelDiscardReplyCallback, NULL, "SLAVEOF %s %s", host, portstr);
+        sentinelDiscardReplyCallback, NULL, "%s %s %s", ri->master->slave_of_command, host, portstr);
```

Comment:
thank you for pointing that out, I've update those changes to use the default command name when the provided value is an empty string.